netweaver/setup/swap_space.sls

```diff
@@ -0,0 +1,37 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if host == node.host %}
+
+{% set swap_file = '/var/lib/swap/swapfile' %}
+
+create_swap_dir:
+  cmd.run:
+  - name: mkdir -p /var/lib/swap
+
+create_swap_file:
```

Comment:
We should not run the command if the file is already there. Add and `unless/onlyIf` option there checking if the file exist

Reply:
Yes, the creates will check this. It will Only run if the file or files specified by creates do not exist.